MembershipApp/models.py

```diff
@@ -0,0 +1,233 @@
+# Create your models here.
+import datetime
+
+from django.contrib.auth.models import User
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.db.models import ImageField
+from django_countries.fields import CountryField
+from phonenumber_field.modelfields import PhoneNumberField
+
+
+def year_choices():
```

Comment:
can this be replaced by regex to validate the data? Generating the list is unneeded compute.